src/tests/ftest/slurm_setup.py

```diff
@@ -60,17 +79,28 @@ def update_config_cmdlist(args):
     # Copy the slurm*example.conf files to /etc/slurm/
     if execute_cluster_cmds(all_nodes, COPY_LIST, args.sudo) > 0:
         sys.exit(1)
-
-    cmd_list = [
-        "sed -i -e 's/ControlMachine=linux0/ControlMachine={}/g' {}".format(
-            args.control, SLURM_CONF),
-        "sed -i -e 's/ClusterName=linux/ClusterName=ci_cluster/g' {}".format(
-            SLURM_CONF),
-        "sed -i -e 's/SlurmUser=slurm/SlurmUser={}/g' {}".format(
-            args.user, SLURM_CONF),
-        "sed -i -e 's/NodeName/#NodeName/g' {}".format(
-            SLURM_CONF),
-        ]
+    if distro == "el8" or distro == "suse.lp153":
```

Comment:
(pylint-consider-using-in) Consider merging these comparisons with "in" to "distro in ('el8', 'suse.lp153')"